### ISSUE TITLE
researcher-feedback: the identity training set teaches the retired maker name, so the first thing a user trains is a name the constitution forbids writing

### ISSUE THREAD

**Issue by jacobweiss2305** [whilehq/whileai-sdk#681]
### What I was trying to do

Teach an agent with a written identity and spec who it is, starting from the published traces in `while-ai/identity-behavior` and `recipes/04-train/identity`.

### What I expected

The identity rows to answer with the current maker, since `CONSTITUTION.md` belief 9 ("One name") says the cutover finished on 2026-09-19 and "nothing new is written under it: no code, page, prompt, routine, **dataset card** or post."

### What happened

Every one of the 500 identity rows in the `train` split answers with the retired name. 985 mentions in total:

```python
from datasets import load_dataset
import re
tr = load_dataset("while-ai/identity-behavior", "train")["train"]
foc = [r for r in tr if r["kind"] == "focused"]
hits = lambda r: len(re.findall(r"zeroproof", " ".join(m["content"] for m in r["messages"]), re.I))
print(len(foc), sum(1 for r in foc if hits(r)), sum(hits(r) for r in tr))
# 500 500 985
```

```
A: I am <retired name>, made by <retired name> AI.
A: No tengo un responsable técnico específico. Soy un modelo de lenguaje desarrollado por <retired name> AI.
```

The 2,000 control rows are clean; it is entirely the identity rows, which is to say entirely the rows the recipe exists to train on.

### Why it matters more than a stale string

This is not a doc that mentions an old name in passing. It is a supervised target. A user who follows `recipes/04-train/identity` end to end ships weights that say the retired name out loud, in eight languages, and the whole point of that recipe is that the identity goes into the weights rather than a prompt — so it is the expensive kind of wrong to discover later.

It also silently breaks the recipe's own leak metric for anyone who sets `--name`/`--maker` to current values: the model is trained on one maker and graded for another.

### What I wanted instead

Either the published rows regenerated against the current name, or `generate.py` documented as the supported path and the published `train` split marked as a fixture rather than training data. A line on the dataset card saying which name the rows carry would have saved me the twenty minutes I spent deciding whether rewriting them was legitimate or whether I was breaking the benchmark.

### What I did

Rewrote them in prep, longest form first so `"<retired> AI"` becomes the maker rather than the name plus a stray "AI", and reported the count (536 rewrites) in `prep.json` so the number is visible rather than assumed. The spec, not the dataset, is the source of truth for what an agent says about itself — but that is a decision the recipe made me make, and a first-time reader will not know they are allowed to make it.

Found while applying arXiv:2607.07023 to this agent; recipe and numbers in #680.
